package.json

```diff
@@ -21,7 +21,10 @@
     "demo:start": "builder concurrent demo:start:app demo:start:graphql-server",
     "type-check": "tsc",
     "lint": "tslint 'src/**/*.{ts, tsx}'",
-    "test": "jest"
+    "test": "jest",
+    "prettier":
+      "prettier --write '{,!(node_modules|custom-typings)/**/}*.{js,jsx,ts,tsx,json,md}'",
```

Comment:
@baer -- I'm checking as soon as my VM boots up, but I thought single quotes didn't play nice on Windows. Did you check there? (I usually use escaped double quotes...)

Reply:
![screen shot 2018-01-30 at 3 48 13 pm](https://user-images.githubusercontent.com/445557/35597811-02507a78-05d5-11e8-8967-a2425f22138e.png)

😿

Reply:
Yep. Not windows compliant.

```
C:\Users\IEUser\Desktop\scm\urql>yarn run prettier
yarn run v1.3.2
$ prettier --write '{,!(node_modules|custom-typings)/**/}*.{js,jsx,ts,tsx,json,md}'
'custom-typings)' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 255.
```

Reply:
```js
"prettier":
      "prettier --write \"{,!(node_modules|custom-typings)/**/}*.{js,jsx,ts,tsx,json,md}\"",
```

works on Windows.